db/migrate/20170730154117_create_account_migrations.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAccountMigrations < ActiveRecord::Migration[5.1]
+  def change
+    create_table :account_migrations do |t|
+      t.integer :old_person_id, null: false
+      t.integer :new_person_id, null: false
+    end
+
+    add_foreign_key :account_migrations, :people, column: :old_person_id
+    add_foreign_key :account_migrations, :people, column: :new_person_id
+
+    add_index :account_migrations, %i[old_person_id new_person_id], unique: true
```

Comment:
> We can remove this index if we introduce this feature afterwards, right?

Yes, we need to remove the index when we add support for receiving merge migrations. Import an export to an existing profile would again be a second step for that. But as said, that's something for later. If we already know we want to support merging later, we can omit the index here, because the rails validation should do it for now. (then we don't need to add the index here and remove it later again).

> Also we have separate `unique` validation for new and old person in Rails now.

As said, I don't trust rails validations ;) They do a select and see if the object exists, and then save it. We already had duplicates with rails validation in the past, because things happened in parallel, both did the select, saw that there is nothing there, and saved it ;) So every unique validation in rails should be covered by the database (even when the risk for this to happen is really small, because it is federated via public receiver, so only once per pod). I'd only make an exception if we know we want to remove that index again. That way support for receive merging could be added in a minor without the need of a migration.